src/ts/components/seekbar.ts

```diff
@@ -48,6 +50,60 @@ export interface SeekBarMarker {
   duration?: number;
 }
 
+enum SeekBarType {
```

Comment:
Can we extract this whole code block into a separate file? Maybe a `SeekBarArrowKeyController` or something. (`seekbar.ts` is already really big (+1.1k LOC))